src/nlp/builder.py

```diff
@@ -165,6 +164,10 @@ def __init__(
                 )
                 os.rmdir(self._cache_dir)
 
+        # update info with user specified infos
+        if info is not None:
+            self.info.update(info)
```

Comment:
Doesn't this break because some values in `DatasetInfo` like `description`, `citation`, etc are empty strings by default and not `None`?

Do we need to give access to other attributes than `features`? Otherwise, I would favor a more robust approach of using directly features which would avoid potential side effect if we change the DatasetInfo default values later.

Reply:
Yes all the values are are not None would overwrite the existing ones, which is not good.
I'm fine with using only features then.